cfgov/v1/config.py

```diff
@@ -0,0 +1,12 @@
+ # These messages are manually mirrored on the
```

Comment:
Lets me this to the `v1/util/util.py` file. its purpose is to house helper methods / configuration related things